third_party/blink/renderer/bindings/core/v8/record_replay_events.cc

```diff
@@ -56,8 +56,8 @@ static void ReplayNotifyAfterEvent(const String& eventName,
 bool ShouldNotifyEvent(const String& eventName) {
   return IsRecordingOrReplaying() && !AreEventsDisallowed() &&
          !eventName.empty() &&
-         // Disabled by default (RUN-1251)
-         !FeatureEnabled("disable-collect-events") &&
+         // check for events feature flag (RUN-1609)
+         FeatureEnabled("collect-events") &&
```

Comment:
This FeatureEnabled call can be combined with the IsRecordingOrReplaying call above.